ansible/roles/test/files/ptftests/arista.py

```diff
@@ -239,19 +244,24 @@ def parse_logs(self, data):
                 assert(events[0][1] != 'Established')
 
             assert(events[-1][1] == 'Established')
+            # TODO assert if the peer down to established time is more than 165s
+
+        # verify establishment time between v4 and v6 peer is not more than 20s
```

Comment:
>between v4 and v6 peer [](start = 36, length = 22)

Could you explain "between v4 and v6 peer" and why the time is useful?

Reply:
no longer having this check. only addressing the log issue in this PR